wikimedia/wikipedia-ios
# Update supported Wikipedia project languages

**Phabricator:** https://phabricator.wikimedia.org/T317844

### Notes
Adds `ami`, `blk`, `dag`, `guw`, `kcg`, `pcm`, `pwn`, `shi` via the `Update Languages` script.

### Test Steps
We can rely on QA for extensive testing (I updated the original task description), but for a couple of the language codes above you can:
1. Search for a language code/language name from above
2. Confirm you can search for/view an article in that language
